README.md

```diff
@@ -108,43 +108,86 @@ See [CONFIG.md](CONFIG.md) for details.
 See [Code Samples](https://github.com/hazelcast/hazelcast-nodejs-client/tree/master/code_samples)
 
 
-# Serialization Considerations
+# Serialization
```

Comment:
Agreeing with Mustafa: We need to have an intro here explaining what is serialization and why do we need it. After those explanations, we also need to give a paragraph about custom serialization including some use cases so that the readers understand when they need a custom serialization.

Reply:
@furkansenharputlu please do not use `custom serialization` as a term describing any serialization that is not Hazelcast native serialization. `custom serialization` has a specific meaning. Custom serializer is one of the serialization methods that the user can configure (along with IdentifiedDataSerializable, Portable and Global Serializer).

What I understand from this section is everything except primitives are converted to JSON by default(which is correct) and the only way to use something else is to configure a custom serializer (which is wrong and not a popular choice at all)

Reply:
I gave a link to `Custom Serialization`.

Reply:
@furkansenharputlu I am not sure how giving a link to custom serialization addresses my concern. The documentation still uses custom serialization as the only alternative to default json behaviour. see `the default behavior can be changed by using a [Custom Serialization](#3.-custom-serialization).`

Reply:
I changed that sentence.